### PR TITLE
Change the `Ping` trait to match the RLB API

### PR DESCRIPTION
**Pull request by Dexterp37**
This also adds a notice to the RLB type to mention to keep the trait updated.

### CODE REVIEW

**Comment by Dexterp37** [comment]
The beta error is being fixed in #1331

File: glean-core/rlb/src/private/ping.rs [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 #[derive(Clone, Debug)]
 pub struct PingType {
     pub(crate) name: String,
     pub(crate) ping_type: glean_core::metrics::PingType,
 }
 
+// IMPORTANT:
+//
+// When changing this implementation, make sure all the operations are
+// also declared in the related trait in `../traits/`.
```
Comment (brizental) [inline glean-core/rlb/src/private/ping.rs:17]: I don't understand this comment, 
1. I couldn't find the traits folder you are pointing to here; 
2. Shouldn't this comment be on line 47 before `impl glean_core::traits::Ping for PingType`? 
3. Wouldn't Rust error anyways if you don't implement a trait function that doesn't include a default?

**Review by brizental — APPROVED** [review]
r+wc

File: glean-core/rlb/src/private/ping.rs (hunk 1 above)
Comment (Dexterp37) [reply glean-core/rlb/src/private/ping.rs:17]: Gah, you're right. This comment should be removed. Copy pasta.

**Pull request merged**